common/core_api_review/kibana.api.ts

```diff
@@ -0,0 +1,72 @@
+// @public
+interface CallAPIOptions {
+    wrap401Errors: boolean;
+}
+
+// @public
+declare class ClusterClient {
+    // (undocumented)
+    constructor(config: ElasticsearchClientConfig, log: Logger);

```

Comment:
Is there any way to ensure that all the types referenced in the config are included in the API? Even though `ElasticsearchClientConfig` isn't exported it's still a part of the "public api" right? Changes to that class should be considered a change to the Public API I would think.

Reply:
I'll follow-up upstream, but I agree this seems like a bug. I also noticed that generating markdown documentation doesn't create a paragraph for a class' constructor (the yaml generator does). I'll keep tabs on or help fix the issues upstream, but don't want to block this PR on that.